tests/dags/test_chain_xcomargs.py

```diff
@@ -0,0 +1,66 @@
+#
```

Comment:
This shouldn't be a new file, but some extra cases with asserts here https://github.com/apache/airflow/blob/fa811057a6ae0fc6c5e4bff1e18971c262a42a4c/tests/models/test_baseoperator.py#L381-L401

Reply:
Added similar cases for `XComArgs`.